frontend/src/modules/Organizations/services/index.js

```diff
@@ -0,0 +1,9 @@
+export * from './archiveOrganization';
+export * from './createOrganization';
+export * from './getOrganization';
+export * from './inviteGroup';
+export * from './listOrganizationEnvironments';
```

Comment:
I agree on placing the `listOrganizationEnvironments` here. It matches the decision in the backend